class ValidationException(Exception):
    pass


class NotFoundException(Exception):
    pass
